lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
@dougwilson That's really odd. Can you confirm that the following `flushHeaders` function produces that error?

```
    try {
      this.res.statusCode = 200;
      this.res.writeHead();
    } catch (e) {
      console.error(e.stack);
    }
```